docs/templates/moved_to_collections.rst.j2

```diff
@@ -0,0 +1,29 @@
+{# avoids rST "isn't included in any toctree" errors for module docs #}
+{% if plugin_type == 'module' %}
+:orphan:
+{% endif %}
+
+.. _@{ module }@_@{ plugin_type }@:
+{% for alias in aliases %}
+.. _@{ alias }@_@{ plugin_type }@:
+{% endfor %}
+
+{% if short_description %}
+{%   set title = module + ' -- ' + short_description | rst_ify %}
+{% else %}
+{%   set title = module %}
+{% endif %}
+
+@{ title }@
+@{ '+' * title|length }@
+
+This module has moved to a collection.  To install it::
+
+    ansible-galaxy collection install @{collection}@
+
```

Comment:
Possibly include a link collection docs here?

Reply:
On galaxy or on docs.ansible.com?  The latter is at the bottom of the new template